apps/server/src/modules/user/controller/user.controller.ts

```diff
@@ -1,13 +1,11 @@
-import { ApiTags } from '@nestjs/swagger';
 import { Body, Controller, Get, Patch } from '@nestjs/common';
-
+import { ApiTags } from '@nestjs/swagger';
 import { ICurrentUser } from '@shared/domain';
 import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator/auth.decorator';
-
+// eslint-disable-next-line import/no-cycle

```

Comment:
Is this line needed? I don't get a linter error in user.controller after removal.

Reply:
Line was removed